src/dialog/overflow-menu/overflow-menu-pane.component.ts

```diff
@@ -35,7 +35,7 @@ export class OverflowMenuPane extends Dialog implements AfterViewInit {
 	}
 
 	onDialogInit() {
-		this.addGap["bottom"] = pos => {
+		this.addGap["bottom"] = this.addGap["top"] = pos => {
```

Comment:
I see what this is doing, but for readability it would be better to create a separate function and just assign it to `this.addGap["top"]`/`this.addGap["bottom"]`

Always go for the more readable option, even if it's a little more typing 😄